src/main/java/com/hobbycollection/dao/CollectionItemDAO.java

```diff
@@ -21,7 +21,7 @@ public CollectionItem getItemByID(int id) {
 
     @Override
     public CollectionItem save(CollectionItem collectionItem) throws Exception {
-        Integer collectionItemId = collectionItem.getID();
+        int collectionItemId = collectionItem.getID();
```

Comment:
I changed the collection item id from integer to int. Because int performs slightly better than Integer.
Sources:
https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3
https://www.geeksforgeeks.org/difference-between-an-integer-and-int-in-java/